cfme/infrastructure/virtual_machines.py

```diff
@@ -1518,3 +1527,40 @@ def VIEW(self):     # noqa
 
     def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.monitoring.item_select('Utilization')
+
+
+@navigator.register(InfraVm, 'PolicySimulation')
+class PolicySimulation(CFMENavigateStep):
+    VIEW = PolicySimulationView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self, *args, **kwargs):
+        self.prerequisite_view.toolbar.policy.item_select('Policy Simulation')
+
+
+@navigator.register(InfraVmCollection, "PolicySimulation")  # noqa
+class PolicySimulation(CFMENavigateStep):  # noqa
```

Comment:
The class itself should have a unique name, but the registered destination name can be the same on both the collection and entity.